packages/bundler/src/modules/BundleManager.ts

```diff
@@ -89,9 +89,16 @@ export class BundleManager {
       const signedTx = await this.signer.signTransaction(tx)
       let ret: string
       if (this.conditionalRpc) {
+        const currentBlockNumber = await this.provider.getBlockNumber();
```

Comment:
why should we use specific block number? 
The beauty sendRawTransactionConditional is that you can collect the "real" required conditions: since a bundle depends on storage (which we provide in storageMap) and can't depend on any other environment info (such as block number, timestamp) because of the validation rules, we don't really care if it is included in the next block or not.

Reply:
This is just an additional application supplement to the Optimism version of txoptions, adding the max/min block number. Of course, we can also pass this information through commands.